satpy/tests/test_composites.py

```diff
@@ -456,20 +456,29 @@ def test_compositor(self, e2d):
         """Test luminance sharpening compositor."""
         from satpy.composites import SandwichCompositor
 
-        rgb_arr = da.from_array(np.random.random((3, 2, 2)), chunks=2)
-        rgb = xr.DataArray(rgb_arr, dims=['bands', 'y', 'x'])
+        # Test RGBA
+        bands = ['R', 'G', 'B', 'A']
+        rgba_arr = da.from_array(np.random.random((4, 2, 2)), chunks=2)
```

Comment:
The issue here is now you've removed the test for RGB data. The easiest thing might be to change this class to a regular class not based on `unittest.TestCase` and use `pytest.mark.parametrize` to run the same test for different kinds of data. Like:

```
class TestSandwichCompositor:

    @pytest.mark.parametrize("input_shape", ((3, 2, 2), (4, 2, 2)))
    @mock.patch('satpy.composites.enhance2dataset')
    def test_compositor(self, e2d, input_shape):
        bands = ['R', 'G', 'B', 'A'][:input_shape[0]]
        data_arr = da.from_array(np.random.random(input_shape), chunks=2)
        ...
```